core/src/raw/chrono_util.rs

```diff
@@ -53,3 +53,12 @@ pub fn parse_datetime_from_from_timestamp_millis(s: i64) -> Result<DateTime<Utc>
 
     Ok(st.into())
 }
+
+/// parse datetime from given timestamp
+pub fn parse_datetime_from_from_timestamp(s: i64) -> Result<DateTime<Utc>> {
```

Comment:
nits: I think `s: u64` would be better?

Reply:
Feel free to submit a PR if you believe it's worthwhile. Let's avoid continuously adding tasks to a single PR.